src/components/atoms/Table.module.css

```diff
@@ -20,7 +20,7 @@
 
 .table [role='columnheader'] > span,
 .table [role='columnheader'] > div {
-  color: var(--color-secondary);
+  color: var(--brand-purple);
```

Comment:
thought(non-blocking): While a descriptive naming like `brand-purple` helps with imagining the color, it does not help with understanding the function of the color. Is it a highlight? Is it the main color? Is it a secondary color for variation? And so on. Ideally, our portals are built so that we just change the actual colors and do not touch the styles at all if we are not adding any new elements to them.

Reply:
I agree on the methodology you are describing, but sometimes it's difficult since the colors are not working well together. This will need a profound rework of all the css files in the project, maybe it's worth to do it in the main MVG portal since this is the repo used as a base for all the other portals. 
The color variable names also need to be changed since they don't reflect the actual color, but I didn't want to globally rename them to avoid cluttering the PR too much, my plan is to do it after the PR approval in a separate one.

Reply:
Agreed and nothing for this PR. I just wanted to raise this point to get some awareness and room to discuss for future improvements.

Reply:
Also agreeing on this as especially in these forks the color name does not represent the actual color. I will create an issue in the main portal.

Edit: Issue [#128](https://github.com/deltaDAO/mvg-portal/issues/128) created